composer.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
could you please remove this file from the project, I think it's unnecessary

Reply:
damn! don't understand how this file always gets pushed! it doesn't exist in my code! :/ sorry! will be removed